GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -62,7 +62,10 @@ public partial class FormPush : GitModuleForm
         private readonly TranslationString _pullActionMerge = new TranslationString("merge");
         private readonly TranslationString _pullRepositoryCaption = new TranslationString("Push was rejected from \"{0}\"");
         private readonly TranslationString _dontShowAgain = new TranslationString("Remember my decision.");
-
+        private readonly TranslationString _useForceWithLeaseInstead =
+            new TranslationString("Force push may overwrite other people's change. Do you want to use the safer force with lease instead?");
```

Comment:
Force push overwrites changes, including those since your last fetch. Do you want to use the safer force with lease instead?